shinysdr/i/blocks.py

```diff
@@ -302,8 +302,8 @@ def __do_connect(self):
         # It would make slightly more sense to use unsigned chars, but blocks.float_to_uchar does not support vlen.
         self.__fft_converter = blocks.float_to_char(vlen=self.__freq_resolution, scale=1.0)
         
-        fft_sink = self.__fft_cell.create_sink_internal(numpy.dtype((numpy.int8, output_length)))
-        scope_sink = self.__scope_cell.create_sink_internal(numpy.dtype(('c8', self.__time_length)))
+        self.__fft_sink = self.__fft_cell.create_sink_internal(numpy.dtype((numpy.int8, output_length)))
+        self.__scope_sink = self.__scope_cell.create_sink_internal(numpy.dtype(('c8', self.__time_length)))
```

Comment:
Why are these being made attributes?

Reply:
Letting Python garbage collect them while they're still in the flowgraph causes crashes now.